scripts/upgrade

```diff
@@ -63,74 +67,63 @@ ynh_clean_setup () {
 # Exit if an error occurs during the execution of the script
 ynh_abort_if_errors
 
-#=================================================
-# CHECK THE PATH
-#=================================================
-
-# Normalize the URL path syntax
-path_url=$(ynh_normalize_url_path "$path_url")
-
 #=================================================
 # STANDARD UPGRADE STEPS
 #=================================================
 # INSTALL DEPENDENCIES
 #=================================================
-
 ynh_script_progression --message="Installing app dependencies"
-ynh_install_app_dependencies $pkg_dependencies
 
+ynh_exec_warn_less ynh_install_app_dependencies $pkg_dependencies
 
 # Now that postgresql is installed, check the db exists and the user is set up
 ynh_script_progression --message="Setting up database"
 if [ -z "$db_pwd" ]; then
 	ynh_psql_test_if_first_run
-	ynh_psql_setup_db "$db_user" "$db_name"
-	db_pwd=$(ynh_app_setting_get "$app" psqlpwd)
+	ynh_psql_setup_db --db_user=$db_user --db_name=$db_name
```

Comment:
We should probably sanitize the db_name before (like on install)? `db_name=$(ynh_sanitize_dbid --db_name=$app)`

Reply:
Why not set it like in example_ynh template ('if [ -z "$db_name" ]; then' ...)
https://github.com/YunoHost/example_ynh/blob/25251b00d316c41ed1fef6df90312df60615b8da/scripts/upgrade#L50-L54

Reply:
This will store the db_name in the app's setting but that won't create it I think…

Reply:
See https://github.com/YunoHost/example_ynh/blob/25251b00d316c41ed1fef6df90312df60615b8da/scripts/upgrade#L46.

Our case is that Kresus 0.15 did not have a postgresql database, so on ugprade we need to create it, not just save the db_name in setting.